packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -152,13 +157,31 @@ class DDSMastheadComposite extends LitElement {
         ${!title
           ? undefined
           : html`
-              <dds-masthead-l1-name title="${title}" url="${url}"> </dds-masthead-l1-name>
+              <dds-masthead-l1-name title="${title}" url="${url}"></dds-masthead-l1-name>
             `}
         ${this._renderL1Items({ target: NAV_ITEMS_RENDER_TARGET.TOP_NAV })}
       </dds-masthead-l1>
     `;
   }
 
+  /**
+   * Renders masthead logo
+   *
+   */
+  private _renderLogo() {
+    if (!this.logoData)
+      return html`
+        <dds-masthead-logo></dds-masthead-logo>
+      `;
+    const useAlternateLogo = MastheadLogoAPI.setMastheadLogo(this.logoData);
+    const { tooltip, svg } = this.logoData;
+    return html`
+      <dds-masthead-logo ?hasTooltip="${ifNonNull(tooltip)}" aria-label="${ifNonNull(tooltip)}"
```

Comment:
Boolean attributes are removed for `null`/`undefined` anyway because those values are falsy:

```suggestion
      <dds-masthead-logo ?hasTooltip="${tooltip}" aria-label="${ifNonNull(tooltip)}"
```